.travis.yml

```diff
@@ -53,6 +53,7 @@ install:
     - travis_retry DOCKER_EXEC apt-get update
     - travis_retry DOCKER_EXEC apt-get install --no-install-recommends --no-upgrade -qq $PACKAGES $DOCKER_PACKAGES
 before_script:
+    - DOCKER_EXEC [ ! -d \$HOME/.bitcoin ]  # Make sure default datadir does not exist
```

Comment:
WDYT about creating a dummy file? It would cause failure if used right?

Reply:
Indeed. That would then give a nice bt in case of failure. Thanks

```yaml
    - DOCKER_EXEC echo \> \$HOME/.bitcoin  # Make sure default datadir does not exist and is never read by creating a dummy file